### ISSUE TITLE
I want to delete /en/ path

### ISSUE THREAD

**Issue by eaykutalp** [psephopaiktes/astro-i18n-starter#1]
prefixDefaultLocale: false,

not working, help me thanks.

**Comment by psephopaiktes** [comment]
Yes, as noted here, we do not currently expect to turn off the `prefixDefaultLocale` setting. 🥲
This is because there is a part of the process that retrieves the user's language from the URL path.
https://astro-i18n-starter.pages.dev/en/setup/

---

However, if the URL does not contain `/[lang]/`, the user is redirected through 404.html.
https://astro-i18n-starter.pages.dev/en/feature#404

For example,
https://astro-i18n-starter.pages.dev/blog/first-post/
will redirect to
https://astro-i18n-starter.pages.dev/en/blog/first-post/ .


---

Thank you for you feedback.

**Comment by eaykutalp** [comment]
Isn't there a way to do this? I'm not a software developer, but I was going to look at your software and install multiple languages ​​in another template.

This is very important for SEO, so the root path must have a language.

**Comment by eaykutalp** [comment]
hello dou you here?

**Comment by psephopaiktes** [comment]
I can't think of a way to fix this problem at this time, as it would require fixing it from the design roots.
Save the Issue as a Future Request.


> SEO

The root path (/index.html) is redirected, but <link rel="alternate"/> and OGP are set for SEO.

Please check it by typing the following in the address bar of Chrome.
view-source:https://astro-i18n-starter.pages.dev/

**Comment by eaykutalp** [comment]
hello, how is completed i can see /en/ path
